docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_YOLACT.md

```diff
@@ -0,0 +1,164 @@
+-
+-# As of March 10, 2019, Pytorch DataParallel still doesn't support JIT Script Modules
+-use_jit = torch.cuda.device_count() <= 1
+-if not use_jit:
+-    print('Multiple GPUs detected! Turning off JIT.')
++use_jit = False
+ 
+ ScriptModuleWrapper = torch.jit.ScriptModule if use_jit else nn.Module
+ script_method_wrapper = torch.jit.script_method if use_jit else lambda fn, _rcn=None: fn
+ 
+ 
++def decode(loc, priors):
++    variances = [0.1, 0.2]
++    boxes = torch.cat((priors[:, :2] + loc[:, :, :2] * variances[0] * priors[:, 2:], priors[:, 2:] * torch.exp(loc[:, :, 2:] * variances[1])), 2)
++
++    boxes_result1 = boxes[:, :, :2] - boxes[:, :, 2:] / 2
++    boxes_result2 = boxes[:, :, 2:] + boxes[:, :, :2]
++    boxes_result = torch.cat((boxes_result1, boxes_result2), 2)
++
++    return boxes_result
++
+ 
+ class Concat(nn.Module):
+     def __init__(self, nets, extra_params):
+@@ -476,7 +479,10 @@ class Yolact(nn.Module):
+     
+     def load_weights(self, path):
+         """ Loads weights from a compressed save file. """
+-        state_dict = torch.load(path)
++        if torch.cuda.is_available():
++            state_dict = torch.load(path)
++        else:
++            state_dict = torch.load(path, map_location=torch.device('cpu'))
+ 
+         # For backward compatability, remove these (the new variable is called layers)
+         for key in list(state_dict.keys()):
+@@ -673,8 +679,11 @@ class Yolact(nn.Module):
+                 else:
+                     pred_outs['conf'] = F.softmax(pred_outs['conf'], -1)
+ 
+-            return self.detect(pred_outs, self)
++            pred_outs['boxes'] = decode(pred_outs['loc'], pred_outs['priors']) # decode output boxes
+ 
++            pred_outs.pop('priors') # remove unused in postprocessing layers
++            pred_outs.pop('loc') # remove unused in postprocessing layers
++            return pred_outs
+ 
+ 
+ 
+-- 
+```
+3. Save and close the file.
+
+## Convert YOLACT Model to the IR
+
+**Step 1**. Clone the GitHub repository and check out the commit:
+
+1. Clone the YOLACT reposirory:
+```sh
+git clone https://github.com/dbolya/yolact
+```
+2. Check out the necessary commit:
+```sh
+git checkout 57b8f2d95e62e2e649b382f516ab41f949b57239
+```
+
+**Step 2**. Download pre-trained model, for example `yolact_base_54_800000.pth`.
+
+**Step 3**. Export model to ONNX* format.
+
+1. Apply the `YOLACT_onnx_export.patch` patch to the repository. Refer to the <a href="#patch-file">Create a Patch File</a> instructions if you do not have it:
+```sh
+ git apply /path/to/patch/YOLACT_onnx_export.patch
+```
+
+2. Evaluate the YOLACT mode in order to export to ONNX* format:
+
+```sh
+python3 eval.py
+    --trained_model=/path/to/yolact_base_54_800000.pth
+    --score_threshold=0.3
+    --top_k=10
+    --image=/path/to/image.jpg
+```
+
+3. You should get `yolact.onnx` file.
+
+**Step 4**. Convert the model to the IR:
+
+```sh
+python path/to/model_optimizer/mo.py --input_model /path/to/yolact.onnx
```

Comment:
what about mean, scale and reverse_input_channles.
In OMz we provide following arguments for conversion:
https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/public/yolact-resnet50-fpn-pytorch/model.yml#L99-L104

But they are model specific (resnet50/resnet101 is the same, darknet53 different).
Mean and scale values can be obtained from model config https://github.com/dbolya/yolact/blob/master/data/config.py

Reply:
We generally don't provide such arguments for model conversion documentation. @lazarevevgeny what is your impression on this issue?

Reply:
as I see they are provided in YOLO, but if it is not so important than ok

Reply:
Regarding comment that we don't have such arguments in documents. In these cases they are probably not needed because the mean/scale are a part of the model itself. My opinion is that we should provide mean/scale values so the customer has full understanding what is going on and how to infer the model.